ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs

```diff
@@ -109,30 +109,30 @@ public int PeriodOfRecord
                 NotifyPropertyChanged(); 
             }
         }
-
         public ObservableCollection<FlowDoubleWrapper> AnalyticalFlows
         {
             get  {return _AnalyticalFlows; }
             set { _AnalyticalFlows = value; NotifyPropertyChanged();}
         }
-        public ObservableCollection<FlowDoubleWrapper> GraphicalFlows
-        {
-            get { return _GraphicalFlows; }
-            set { _GraphicalFlows = value; NotifyPropertyChanged(); }
-        }      
+        public TableWithPlotVM MyGraphicalVM { get; set; }
 
         #endregion
         #region Constructors
+        //This supports a fresh editor
         public AnalyticalFrequencyEditorVM(ComputeComponentVM defaultCurve,  EditorActionManager actionManager) : base(defaultCurve, actionManager)
```

Comment:
does it matter that this is within the analytical frequency editor?

Reply:
This is bad nomenclature. I'd like to rename this editor, as the same editor supports both graphical and analytical, despite being called analytical. It is the same in the UI. I've made a ticket for this. https://github.com/HydrologicEngineeringCenter/HEC-FDA/issues/387